batchflow/models/tf/nn/train.py

```diff
@@ -0,0 +1,96 @@
+""" Helpers for training """
+from math import pi
+
+import tensorflow as tf
+from tensorflow.math import sin, asin, floor # pylint: disable=import-error
+
+
+def piecewise_constant(global_step, *args, **kwargs):
+    """ Constant learning rate decay (uses global_step param instead of x) """
+    return tf.train.piecewise_constant(global_step, *args, **kwargs)
+
+def cyclic_learning_rate(learning_rate, global_step, max_lr=0.1, step_size=10,
+                         mode='triangular', name='CyclicLearningRate'):
+    """ Applies cyclic learning rate (CLR).
+    https://arxiv.org/abs/1506.01186.
```

Comment:
Use common style for mentioning publications. See `TFModel.se_block` for example.